openedx/core/djangoapps/credit/urls.py

```diff
@@ -0,0 +1,18 @@
+"""
+Defines the URL routes for this app.
+"""
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from openedx.core.djangoapps.credit.tests.fake_update_min_grade import UpdateMinRequirementFakeView
+
+urlpatterns = patterns(
```

Comment:
When feature flag is off then urls throws error urlconf %s doesn't have any patterns in it. Thats' why i have defined empty pattern.